Makefile

```diff
@@ -1,49 +1,52 @@
+all: insert_mysql_ex delete_mysql_ex update_mysql_ex mysqlex pgsql insert_pg_ex delete_pg_ex \
```

Comment:
Hopefully a lot of this can go away once we migrate this to being a mason package. Q - does Spack have all the packages we depend on?